early-init.el

```diff
@@ -4,7 +4,7 @@
 
 ;;; Code:
 
-;; We use `straight.el' for package management, so we disable
+;; We use `elpaca.el' for package management, so we disable
 ;; `package.el' at startup to avoid loading packages twice.
 (setq package-enable-at-startup nil)
```

Comment:
I'm editing the comment, but I'm not sure about it. Wouldn't it be good to set it to `t`?

Reply:
This has to do with another built-in packaging system for emacs called `package.el` that leaves a lot to be desired. We don't use it anywhere, so it off is safe, and really speeds up load times.